README.md

```diff
@@ -30,7 +30,7 @@ Get a supported CMSSW release area:
 Checkout the FinalStateAnalysis repository:
 
 ```bash
-  git clone --recursive -b miniAOD_dev https://github.com/uwcms/FinalStateAnalysis.git
+  git clone --recursive -b miniAOD_dev git@github.com:uwcms/FinalStateAnalysis.git
```

Comment:
Because most people around here seem to use git with SSH, so we may as well get rid of the warning message that appears if you use https. If I'm wrong and more people use HTTPS, I can change it back.